darts/models/forecasting/block_rnn_model.py

```diff
@@ -195,6 +210,7 @@ def __init__(
         n_rnn_layers: int = 1,
         hidden_fc_sizes: Optional[List] = None,
         dropout: float = 0.0,
+        activation: Optional[str] = None,
```

Comment:
I'm thinking whether it'd be better to just use a default activation:

- If there are no hidden layers, then no activiation will be added in any way.
- In case of hidden layers, then we use the default.

Otherwise every time you want to use `hidden_fc_sizes` you'd have to also specify the activiation function which might be annoying.

@madtoinou what do you think?

```suggestion
        activation: str = "ReLU",
```